config/_default/menus.yml

```diff
@@ -107,7 +107,7 @@ reference:
     url: https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v3/go/pulumi
     weight: 3
 
-  - name: .NET Core
+  - name: .NET
```

Comment:
This is good, but these config files only apply to their local repositories; Hugo doesn’t bring them across when it imports a module, so we’ll need to make this same change in the docs repo as well. (Sorry about that — one of those things.)

Reply:
To be sure - do I need to open a PR to change this file https://github.com/pulumi/docs/blob/master/config/_default/menus.yml? It looks like that file is actually newer - has Java and YAML - do we need to sync other things?